src/Type.cpp

```diff
@@ -1911,7 +1928,11 @@ void OutputStructUnion(Type* type, std::ostream &out)
 			OutputUnionAssignOps(type, out, true);
 		}
 
-        out << "};";
+        out << "}";
+	if(type->eType == eStruct || type->eType == eUnion){
```

Comment:
Why do we limit to structs and unions? `deprecated` and `unused` should apply to all types, while `transparent_union` should apply to just unions.

Reply:
No. The PR is generating `deprecated`, `unused`, etc for simple and pointer type also. It is being fuzzed from the `Variable` class. Please check in Variable attributes commit.

I have pushed changes by which `transparent_union` will be applied to just unions